pkgs/development/libraries/qt-5/5.8/qtbase/default.nix

```diff
@@ -15,7 +17,8 @@
   cups ? null, mysql ? null, postgresql ? null,
 
   # options
-  mesaSupported, mesa,
+  mesaSupported ? (!stdenv.isDarwin),
```

Comment:
Thx for the insights @vcunat . I still propose to re-enable mesa/libGL support in a separate PR for the series of currently maintained qt-5 release. What do you think?